content/blog/2015-04-23-prometheus-posts-around-the-web.md

```diff
@@ -0,0 +1,59 @@
+---
+title: "Prometheus Posts around the Web"
+created_at: 2015-04-23
+kind: article
+author: Brian Brazil
+---
+
+It has been almost three months since Prometheus version 0.10.0 was publicly announced and we're now at version 0.13.1.
+
+[SoundCloud's post](https://developers.soundcloud.com/blog/prometheus-monitoring-at-soundcloud)
+remains the best overview of the key components of Prometheus, but there have
+been lots of other online activity around Prometheus. This post will let you catch up on anything you missed.
```

Comment:
Reading over the intro again, how is this for a slight optimization (active case in first sentence, slight grammar correction, add sentence about intention of this blog, ...):

```
It has been almost three months since we publicy announced Prometheus version
0.10.0, and we're now at version 0.13.1.

[SoundCloud's announcement blog post](https://developers.soundcloud.com/blog/prometheus-monitoring-at-soundcloud)
remains the best overview of the key components of Prometheus, but there has
been a lot of other online activity around Prometheus. This post will let you
catch up on anything you missed.

In the future, we will publish more articles and announcements about Prometheus
in this blog.
```